tests/unit/components/PhoneNumberInput.spec.js

```diff
@@ -1,20 +1,59 @@
-import { mount } from '@vue/test-utils';
-import Component from '../../../src/components/PhoneNumberInput.vue';
+import { mount } from "@vue/test-utils";
+import PhoneNumberInput from "../../../src/components/PhoneNumberInput.vue";
 
-describe('PhoneNumberInput.vue', () => {
-  it('renders', () => {
-    const wrapper = mount(Component);
+describe("PhoneNumberInput.vue", () => {
+  it("renders", () => {
+    const wrapper = mount(PhoneNumberInput);
     expect(wrapper.element).toBeDefined();
   });
 });
 
-describe('PhoneNumberInput getCypressValue()', () => {
-  it('contains cypress Value', () => {
-    const wrapper = mount(Component, {
+describe("PhoneNumberInput getCypressValue()", () => {
+  it("contains cypress Value", () => {
+    const wrapper = mount(PhoneNumberInput, {
       props: {
-        cypressId: 'potato'
-      }
+        cypressId: "potato",
+      },
     });
-    expect(wrapper.find("[data-cy=potato]").exists()).toBe(true)
+    expect(wrapper.find("[data-cy=potato]").exists()).toBe(true);
+  });
+});
+
+describe("PhoneNumberInput event handling", () => {
+
+  it("works correctly with v-model", async () => {
+    const wrapper = mount({
+      data() {
+        return { phone: "2" };
+      },
+      template: '<div> <phone v-model="phone" /></div>',
+      components: { phone: PhoneNumberInput },
```

Comment:
Can you keep the element tag name the same as the component name here? So `<PhoneNumberInput />`, and `components: { PhoneNumberInput }`.

This will help make the tests easier to read and understand.